views/help.twig

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.twig' %}
+
+{% block body %}
+  <div class="container">
+    <div class="row" style="padding-top: 70px;">
+      <div id="markdown" class="col">
+        {{ html | raw }}
+      </div>
+    </div>
+  </div>
+{% endblock %}
+
+{% block scripts %}
+  {{ parent() }}
+  <script type="text/javascript" src="./routes/help.js"></script>
```

Comment:
I don't think you meant to serve the route file here hehe ;) You can remove this block, it's not needed :)

Reply:
And could you move the stylesheet block above the body block? Just to keep consistent with the other twig files :)